src/sentry/static/sentry/app/components/charts/baseChart.jsx

```diff
@@ -124,6 +127,7 @@ class BaseChart extends React.Component {
     yAxis: {},
     isGroupedByDate: false,
     interval: 'day',
+    utc: false,
```

Comment:
Should the default utc value be the same as the date time selector? (which is true)